pkg/services/rpcsrv/server.go

```diff
@@ -921,7 +917,7 @@ func (s *Server) calculateNetworkFee(reqParams params.Params) (any, *neorpc.Erro
 				p.Type.EncodeDefaultValue(inv.BinWriter)
 			}
 			if inv.Err != nil {
-				return nil, neorpc.NewInternalServerError(fmt.Sprintf("failed to create dummy invocation script (signer %d): %s", i, inv.Err.Error()))
+				return nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidSignature, fmt.Sprintf("failed to create dummy invocation script (signer %d): %s", i, inv.Err.Error()))
```

Comment:
1. The proposal is correct, since C# node doesn't have neo-project/neo#2805 implemented and will return `ErrInvalidSignature` in some cases.
2. But this particular case in our code is an `InternalServerError`.